charms/jupyter-ui/src/resources.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2021 Canonical
```

Comment:
This could be done with out library [kuberentes-rapper](https://github.com/canonical/kubernetes-rapper). We want to use this library to avoid having these files with the same functionality in each charm.